easysid.cpp

```diff
@@ -152,42 +157,68 @@ EXPORT void set_filter_res(uint8_t res)
     sid.write(23, reg);
 }
 
-EXPORT void set_filter_enable(uint8_t voice)
+EXPORT void set_filter(uint8_t voice, bool enable)
 {
     if (voice < 1 || voice > 3) {
         return;
     }
     char reg = sid.read_state().sid_register[23];
-    reg &= ~0x0f;
-    reg |= 1 < (voice - 1);
+    reg8 mask = (1 << (voice - 1));
+    reg &= ~mask
+    if (enable) {
+        reg |= mask;
+    }
+    sid.write(23, reg);
+}
+
+EXPORT void set_filter_external(bool enable)
+{
+    char reg = sid.read_state().sid_register[23];
+    reg8 mask = (1 << 3);
```

Comment:
You don't need to make a local variable for this. It's a constant. It equals `0x80`. Just use that value instead of `mask`.